vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -91,12 +91,6 @@
       state: present
       kubeconfig: "/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml"
 
-  # Configure Ironic configmap
-  - name: Configure Ironic Configmap
-    shell: |
-      cp {{ BMOPATH }}/ironic-deployment/keepalived/ironic_bmo_configmap.env {{ BMOPATH }}/ironic-deployment/keepalived/ironic_bmo_configmap.env.orig
-      cp {{ IRONIC_DATA_DIR }}/ironic_bmo_configmap.env  {{ BMOPATH }}/ironic-deployment/keepalived/ironic_bmo_configmap.env
-
```

Comment:
I thought this is the piece of code I added to exactly tackle the issue you are trying to solve. Check this https://github.com/metal3-io/metal3-dev-env/pull/554

Reply:
It is not quite the same thing I think. The configmap was working correctly as far as I know, but the other manifests were not. I removed these lines to simplify the process. Like this

Old process:
1. In 03_launch_mgmt_cluster.sh, generate configmaps, modify manifests and deploy BMO/Ironic (here we have the issue that the manifests are not modified for Ironic when using Kind).
2. Clean up (but keep backup of configmap)
3. In move.yml, put the backup of the configmap back. Pivot.
4. Clean up again
5. In move_back.yml, pivot again (but here I think we are missing the correct configmap).

New process:
1. In 03_launch_mgmt_cluster.sh: generate configmaps and modify manifests (also when using kind).
2. Leave everything the way it is. This is simpler and safer than changing and changing back everything.
3. In move.yml, just pivot. No need to change the configmaps or manifests. Everything is already correct.
4. In move_back.yml. Same.

If we really want to clean the repo I think we should add that in the last step in `cleanup.yml`. But my 2 cents is that it is easier to understand what is happening if we let the repo remain dirty. The next time we run the 03 script it will anyway reset or we can manually `git reset --hard`.

Reply:
We have always had a confusion if we should keep it dirty or we should keep it intact and we somehow kept it so that we would keep the files intact. If you check the conversation on the PR I mentioned you will get some idea.

Reply:
+1 for simplifying the bmo ironic configuration steps by removing the repeated `creation and restoration of backups` and move cleaning the repo to the cleanup script or using a make phony where we can do the back up only once at the beginning and restore after the end of the test